### DISCUSSION TITLE
client.IsSessionRegistered is missing

### DISCUSSION THREAD

**Discussion by ssiyad** [xelaj/mtproto#83] [Q&A] [answered]
Is there really a method called `client.IsSessionRegistered` (https://github.com/xelaj/mtproto/blob/main/examples/auth/main.go#L49
)? Can not find that anywhere.

**Comment by quenbyako** [comment, marked as answer]
ehmmm... yes? [Is this line exact what are you looking for?](https://github.com/xelaj/mtproto/blob/216789b95a5d644ebbdd1acb7eb46ff61647960a/telegram/common.go#L128)

**Reply by ssiyad** [reply]
`client.IsSessionRegistered undefined (type *telegram.Client has no field or method IsSessionRegistered)`

this is what I get

**Reply by ssiyad** [reply]
btw, Isn't that function name a typo?

**Reply by quenbyako** [reply]
But i literally paste here link to line, where this method is defined, doublecheck that you didn't delete anything

**Reply by ssiyad** [reply]
Damn it isn't in the source code!
This is in `go.mod`: `github.com/xelaj/mtproto v1.0.0`
If it's an outdated version, how do I upgrade?

**Reply by quenbyako** [reply]
ehm... `go get -u github.com/xelaj/mtproto`

**Reply by ssiyad** [reply]
Ugh, no changes!

**Reply by ssiyad** [reply]
Is it because there is no releases after January?

**Reply by rxw1** [reply]
Maybe I'm late to the party, but `IsSessionRegistred` is not the same as `IsSessionRegistered`
The former, as used in the auth example, is missing an `e`

**Reply by quenbyako** [reply]
Wait a minute...

**Reply by TanyaEleventhGoddess** [reply]
> ehmmm... yes? [Is this line exact what are you looking for?](https://github.com/xelaj/mtproto/blob/216789b95a5d644ebbdd1acb7eb46ff61647960a/telegram/common.go#L128)

Hi, could you please bump the version to solve all these problems right away? The repo is stale from june, plus if I publicly share code using your lib I really wanna avoid including instructions telling to manually update the lib to the latest commit

**Comment by jbub** [comment]
Problem is that README suggests to use `go get github.com/xelaj/mtproto` which will get the latest version which is `v1.0.0` at this point. But `IsSessionRegistered` is not available in that version. `go get github.com/xelaj/mtproto@main` will fetch the latest code where `IsSessionRegistered` is available.

**Reply by boombick** [reply]
@jbub thank you so much!
